### PR TITLE
BOM-1558: Added python 3.8 in test matrix

### PR DESCRIPTION
**Pull request by iamsobanjaved**
**JIRA:** [BOM-1558](https://openedx.atlassian.net/browse/BOM-1558)

### CODE REVIEW

**Comment by awais786** [comment]
@iamsobanjaved add this constraint

# Stay on an LTS release
django<2.3

File: requirements/constraints.txt [hunk 1]
```diff
@@ -14,5 +14,5 @@ futures ; python_version == "2.7"
 # jsonfield2>3.0.3 has dropped support for python 3.5
 jsonfield2==3.0.3
 
-# Version 4.0.0 dropped support for Django < 2.0.1
-django-model-utils<4.0.0
+# pip20+ requires for greater version of pip-tools. But installing 19.3.1 in all our containers and VMs.
```
Comment (awais786) [inline requirements/constraints.txt:17]: Add 1 more constraint.
```
# Stay on an LTS release
django<2.3
```

File: tox.ini [hunk 2]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py{35}-django{111,20,21,22}
+envlist = py{35,38}-django{22,30}
```
Comment (jmbowman) [inline tox.ini:2]: Here also, Django 3.0 with Python3.5 is an invalid combination.

**Comment by iamsobanjaved** [comment]
> Planning to bump the version for a release also?

@jmbowman This is failing on python 3.8. Any need for releasing this?

**Comment by awais786** [comment]
> > Planning to bump the version for a release also?
> 
> @jmbowman This is failing on python 3.8. Any need for releasing this?

@iamsobanjaved approved but wait for this question.

**Pull request merged**